benchalerts/benchalerts/message_formatting.py

```diff
@@ -82,11 +82,35 @@ def _intro_sentence(full_comparison: FullComparisonInfo) -> str:
     return intro + "\n\n"
 
 
-def github_check_summary(full_comparison: FullComparisonInfo) -> str:
+def github_check_summary(
+    full_comparison: FullComparisonInfo, build_url: Optional[str]
+) -> str:
     """Generate a Markdown summary of what happened regarding errors and regressions."""
     summary = _intro_sentence(full_comparison)
 
-    if full_comparison.benchmarks_with_errors:
+    if not full_comparison.has_any_contender_runs:
+        summary += _clean(
+            """
+            None of the specified runs were found on the Conbench server.
+            """
+        )
+        if build_url:
+            summary += f" See the [build logs]({build_url}) for more information."
```

Comment:
Interesting not so important language question: what do you think is nicer?

"see build logs" vs "see the build logs"

I would often choose "see build logs" because it is shorter, but does it also sound like a weird person talking? :D